test/integration_test.js

```diff
@@ -17,86 +17,99 @@ const properties = {
 };
 let clientService;
+let cassandraClient;
 
 describe('Integration test on ClientService', async () => {
-  const mockedFunctionId = 'TestFunction';
+  const mockedByteContract = fs.readFileSync(
+      __dirname + '/StateUpdater.class');
+  const mockedByteFunction = fs.readFileSync(
+      __dirname + '/TestFunction.class');
+  const mockedFunctionId = `TestFunctions${Date.now()}`;
   const mockedContractId = `StateUpdater${Date.now()}`;
   const mockedContractName = 'com.org1.contract.StateUpdater';
   const mockedFunctionName = 'com.org1.function.TestFunction';
-  const mockedAssetId = 'mockedAssetId';
+  const mockedAssetId = `mockedAssetId${Date.now()}`;
   const mockedState = 1;
   const mockedContractArgument = {
     asset_id: mockedAssetId,
     state: mockedState,
   };
-  const mockedFunctionArgument = {
-    asset_id: mockedAssetId,
-    state: mockedState,
-    _function_: mockedFunctionId,
-  };
-  const property = {
+  const contractProperty = {
     properties: 'bar',
   };
-  const functionArgument = {
-    asset_id: mockedAssetId,
-    state: mockedState,
-  };
-  const mockedByteContract = fs.readFileSync(
-      __dirname + '/StateUpdater.class');
-  const mockedByteFunction = fs.readFileSync(
-      __dirname + '/TestFunction.class');
   clientService = new ClientService(properties);
   describe('registerCertificate', () => {
-    it('should works as expected', async () => {
+    it('should return 200 when correct inputs are specified', async () => {
       const response = await clientService.registerCertificate();
       assert.equal(response.getStatus(), 200);
     });
   });
   describe('registerFunction', () => {
-    it('should works as expected', async () => {
+    it('should return 200 when correct parameters are specified', async () => {
```

Comment:
Are you using `parameters` and `inputs` in a different way ? or they are meaning the same ?

Reply:
For me i thought `parameters` is more appropriate because i want to refer to the `(mockedFunctionId, mockedFunctionName, mockedByteFunction)` in `clientService.registerFunction(mockedFunctionId, mockedByteFunction);`. For me `inputs` might be too general...thought?

Reply:
I'm asking because you use both. (using inputs for `registerCertificate` and parameters for some others.)
So If they are used interchangeably here, please make them consistent. If you are using them for some reason, please clarify.

Reply:
In the end i think i'll stick to `inputs`